Apps/ValidationTests/Shared/TestUtils.h

```diff
@@ -112,6 +117,47 @@ namespace Babylon
 #endif
         }
 
+        struct Image
+        {
+            Image() = default;
+            ~Image()
+            {
+                if (m_Image)
+                {
+                    bimg::imageFree(m_Image.get());
+                }
+            }
+            std::unique_ptr<bimg::ImageContainer> m_Image;
```

Comment:
This is a bit confusing. Why do you need a `unique_ptr` when you are `new`ing the `Image`? `m_Image` will get deleted via the default allocator which calls delete `m_Image`. I don't think you want that?